test/integration/ScheduleCreateIntegrationTest.js

```diff
@@ -186,6 +186,360 @@ describe("ScheduleCreate", function () {
+                .execute(env.client)
+        ).getReceipt(env.client);
+
+        let info = await new ScheduleInfoQuery()
+            .setScheduleId(scheduleId)
+            .execute(env.client);
+
+        // Verify the transaction is not executed
+        expect(info.executed).to.be.null;
+
+        // Sign with one key
+        await (
+            await (
+                await new ScheduleSignTransaction()
+                    .setScheduleId(scheduleId)
+                    .freezeWith(env.client)
+                    .sign(key1)
+            ).execute(env.client)
+        ).getReceipt(env.client);
+
+        info = await new ScheduleInfoQuery()
+            .setScheduleId(scheduleId)
+            .execute(env.client);
+
+        // Verify the transaction is still not executed
+        expect(info.executed).to.be.null;
+
+        // Update the signing requirements
+
+        await (
+            await (
+                await (
+                    await (
+                        await new AccountUpdateTransaction()
+                            .setAccountId(accountId)
+                            .setKey(key3)
+                            .freezeWith(env.client)
+                            .sign(key1)
+                    ).sign(key2)
+                ).sign(key3)
+            ).execute(env.client)
+        ).getReceipt(env.client);
+
+        info = await new ScheduleInfoQuery()
+            .setScheduleId(scheduleId)
+            .execute(env.client);
+
+        // Verify the transaction is still not executed
+        expect(info.executed).to.be.null;
+
+        // Sign with the updated key
+        await (
+            await (
+                await new ScheduleSignTransaction()
+                    .setScheduleId(scheduleId)
+                    .freezeWith(env.client)
+                    .sign(key3)
+            ).execute(env.client)
+        ).getReceipt(env.client);
+
+        info = await new ScheduleInfoQuery()
+            .setScheduleId(scheduleId)
+            .execute(env.client);
+
+        // Verify the transaction is executed
+        expect(info.executed).to.not.be.null;
+    });
+
+    it("should execute with short expiration time", async function () {
+        const key = PrivateKey.generateED25519();
+        const hasJitter = false;
+        const SHORT_EXPIRATION_TIME = 10_000;
+
+        const { accountId } = await (
+            await new AccountCreateTransaction()
+                .setKey(key)
+                .setInitialBalance(new Hbar(10))
+                .execute(env.client)
+        ).getReceipt(env.client);
+
+        const transfer = new TransferTransaction()
+            .addHbarTransfer(accountId, new Hbar(1).negated())
+            .addHbarTransfer(env.operatorId, new Hbar(1));
+
+        var { scheduleId } = await (
+            await transfer
+                .schedule()
+                .setExpirationTime(
+                    Timestamp.generate(hasJitter).plusNanos(15_000_000_000),
```

Comment:
This is 15 secs right?
We are sleeping for 10 secs and i think this difference might make this test flaky.